backend/src/chats/message/message.module.ts

```diff
@@ -1,11 +1,17 @@
-import { Module } from '@nestjs/common';
+import { forwardRef, Module } from '@nestjs/common';
 import { MessageService } from './message.service';
 import { MessageController } from './message.controller';
 import { TypeOrmModule } from '@nestjs/typeorm';
 import { Message } from './entities/message.entity';
+import { AuthModule } from '../../auth/auth.module';
+import { SharedModule } from '../../shared/shared.module';
 
 @Module({
-	imports: [TypeOrmModule.forFeature([Message])],
+	imports: [
+		TypeOrmModule.forFeature([Message]),
+		forwardRef(() => AuthModule),
+		SharedModule,
```

Comment:
so here I only need the AuthModule (without forwardRef), not the SharedModule... (no idea how this works)

Reply:
I can remove this for this branch as well, but do you need it in your chat branch? Then I'll leave it in...